packages/@ama-sdk/schematics/schematics/ng-add/index.ts

```diff
@@ -79,7 +80,7 @@ const createOpenApiToolsConfig: Rule = (tree) => {
   const openApiDefaultStorageDir = '.openapi-generator';
   if (tree.exists(openApiConfigPath)) {
     // eslint-disable-next-line @typescript-eslint/naming-convention
-    const openapitoolsConfig = tree.readJson(openApiConfigPath) as { 'generator-cli'?: { storageDir?: string; version?: string } } || {};
+    const openapitoolsConfig = (tree.readJson(openApiConfigPath) || {}) as any as OpenApiToolsConfiguration;
```

Comment:
`{}` is not a value `OpenApiToolsConfiguration`

Reply:
yes, also, probably unnecessary